vaadin-select-flow-demo/src/main/java/com/vaadin/flow/component/select/SelectView.java

```diff
@@ -89,6 +91,30 @@ private void disabledAndReadonly() {
         addCard("Disabled and read-only", layout);
     }
 
+    private void helperText() {
+        // begin-source-example
+        // source-example-heading: Helper text and component
+        Select<String> degree = new Select<>("Baccalaureate", "Licence",
+              "Master", "Doctorate");
+        degree.setLabel("Academic degree");
+        degree.setHelperText(
+              "Please, select only the highest accomplished degree");
+
+        Select<String> pet = new Select<>("Cat", "Dog", "Rabbit", "Fish",
+              "Bird", "Other");
+        pet.setLabel("Pets");
+        pet.setHelperComponent(new Span("Your favorite pet"));
+        pet.getElement().getThemeList().set("helper-above-field", true);
```

Comment:
I think it's best to keep the theme variants in a separate demo as done with other variants and components.

Reply:
Moved to separate demo